### ISSUE TITLE
[BUG]: "Standard" profile limits Winget CLI Functionality

### ISSUE THREAD

**Issue by wgetgoose** [undergroundwires/privacy.sexy#142]
### Description

When using the standard profile, the functionality for "winget" cli's repo is removed. This is possibly due to a blockage of the `https://winget.azureedge.net/cache` URL. 

### OS
Windows 11 Home- Build 22000.708

### Reproduction steps
Use standard profile on privacy.sexy, and attempt to run `winget upgrade --all` or `winget source reset --force`. The repositories will not be able to be updated.

### Scripts
On Hastebin: [https://hastebin.skyra.pw/civuqupoti.bash](url)

### Additional information
What option would I need to disable to restore functionality? I assume it's a telemetry issue, but there are multiple options not specifying winget.

**Comment by Remyie** [comment]
I encountered the same problem. "Failed in attempting to update the source: winget"
Sometimes it also says "Installation failed: system failed to register the windows.firewall extension". Turns out Winget and it's repos require windows defender firewall to be enabled. I believe privacysexy doesn't disable windows defender by default anymore, but if you are not careful enough and decide to both tick "disable windows defender firewall" and "Disable OS components for Defender", it will break winget and several other stuff.

Here's how I fixed it:
- Go to https://privacy.sexy/.
- Privacy Over Security > Disable Windows Defender, click revert on "Disable Windows Defender Firewall" and "Disable OS Components For Defender".
- Download the batch file and run it with admin privileges.
- Restart your PC.
- Open up Powershell and run "winget source reset --force".
- Then run "winget source update".
 
It should be able to update now. If it still says it couldn't update try these:

- Run command prompt as administrator and copy paste these commands:
reg add HKEY_LOCAL_MACHINE\SYSTEM\ControlSet001\Services\mpssvc\ /v Start /t Reg_DWORD /d 2 /f
reg add HKEY_LOCAL_MACHINE\SYSTEM\ControlSet001\Services\mpsdrv\ /v Start /t Reg_DWORD /d 3 /f
- Restart pc.
- Open up Powershell and run "winget source reset --force".
- Then run "winget source update".

If it's still not working try changing your dns.

So in short, it's not a bug with privacysexy, just winget being annoying. Not telling why it's unable to update the repos.
To avoid people accidentally breaking winget, the names of these settings could be changed to include "(Breaks Winget)"

**Comment by undergroundwires** [comment]
Hi @Remyie,

Thank you for all information, sorry for my late response, I missed this issue.

It's addressed since [0.13.2](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.13.2) 🚀.

Actions taken:

1. Winget is specifically mentioned as being broken in script titles:

![image](https://github.com/undergroundwires/privacy.sexy/assets/15555035/122dce9d-09bc-468f-a044-92162513c63b)

2. Winget behavior is documented in script/parent category documentation:

![image](https://github.com/undergroundwires/privacy.sexy/assets/15555035/15f3819a-fc2f-40e0-b7fd-5fd35e4753fe)
